code/datums/religions.dm

```diff
@@ -468,19 +469,19 @@
 	female_adept = "Stockholder"
 	keys = list("capitalism", "free market", "liberalism", "money")
 
-/datum/religion/capitalism/misc(var/mob/living/carbon/human/H)
+/datum/religion/capitalism/equip_chaplain(var/mob/living/carbon/human/H)
 	H.equip_or_collect(new /obj/item/clothing/head/that(H), slot_head)
 
 /datum/religion/america
 	name = "American Exceptionalism"
-	deity_name = "Goerge Washington"
+	deity_name = "Georges Washington"
```

Comment:
Anybody with experience in American history able to confirm whether there were indeed multiple George Washingtons?

Reply:
I think the question should be, which George's Washington?

Reply:
Reee
Google told me it was a valid orthography